src/inputs/input/input.jsx

```diff
@@ -29,46 +50,133 @@ const propTypes = {
      * Deprecated prop. Please use `disable` instead.
      */
     disabled: PropTypes.bool,
+    /**
+     * Indicates that the input has an error.
+     */
     error: PropTypes.oneOfType([
         PropTypes.bool,
         PropTypes.string,
     ]),
+    /**
+     * An input can take on the size of its container.
+     */
     fluid: PropTypes.bool,
+    /**
+     * Tells the component whether to be in guide or no guide mode.
```

Comment:
Maybe:
```
Indicates whether or not the <Input> should be in guide mode.
```

Maybe even some more explanation?  I don't actually know what "guide mode" is off the top of my head! =)

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef

Reply:
Looks good.  Resolving.